js/ftx.js

```diff
@@ -3193,4 +3194,55 @@ module.exports = class ftx extends Exchange {
             'info': info,
         };
     }
+
+    async fetchOpenInterest (symbol, params = {}) {
+        /**
+         * @method
+         * @name ftx#fetchOpenInterest
+         * @description Retrieves the open interest of a currency
+         * @see https://docs.ftx.com/#get-future-stats
+         * @param {string} symbol Unified CCXT market symbol
+         * @param {object} params exchange specific parameters
+         * @returns {object} an open interest structure{@link https://docs.ccxt.com/en/latest/manual.html#interest-history-structure}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
```

Comment:
Should we verify whether market is swap here?

eg:
```js
if (!market['swap']) {
    throw new BadRequest ('Open interest only exist for swap contracts');
}
```

Reply:
yes, this is incorrect, it should check if market is contract instead